supervisor/store/git.py

```diff
@@ -86,16 +109,19 @@ async def clone(self):
                 git.GitCommandError,
             ) as err:
                 _LOGGER.error("Can't clone %s repository: %s.", self.url, err)
-                await self._remove()
-                return False
-
-            return True
+                self.sys_resolution.create_issue(
```

Comment:
Why create an issue instead of just removing the bad repo as before?

Reply:
Removing repo does not solve/fix anything. After I'm finished with the conference stuff, I create something like evolution but to apply fixes. So I can fix it right after start if the internet is back etc... Would be great stuff. Based on sentry, that code never triggered in last 6month

Reply:
I was worried that it would impact the `update_repositories` method that is called when a user add a repo in the UI, but it's not :+1: